lib/viking/model/class_properties/where.js

```diff
@@ -1,7 +1,7 @@
 // Returns a unfetched collection with the predicate set to the query
 Viking.Model.where = function(options) {
     // TODO: Move to modelName as well?
-    var Collection = (this.modelName.className + 'Collection').constantize();
+    var Collection = (this.modelName.name + 'Collection').constantize();
```

Comment:
maybe move this to the Name object? `this.modelName.collection()` which can cache it